packages/simcore-sdk/src/simcore_sdk/node_ports/filemanager.py

```diff
@@ -297,3 +297,25 @@ async def upload_file(
                 return store_id, e_tag
 
     raise exceptions.S3InvalidPathError(s3_object)
+
+
+async def is_metadata_for_entry(store_id: str, s3_object: str) -> bool:

```

Comment:
Q: i wonder if this coroutine can be cancelled because you are capturing with a broad-except. See this example:

```python

async def foo():
  ...
  await is_metadata_for_entry(..)
  ...
  
# somewhere else, a code calls to cancel task associated to foo
task_foo.cancel()
await task_foo  
```